Subalakshmi/scope.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+<body>
+<p>The scope chain in js</p>
+<script>
+var shop= 'shop';
```

Comment:
use proper variable type according to the version of ECMA script you are using.

Reply:
ok ,i will adapt  .. I using let and const variable type in my upcoming programming.